docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.8'
+
+services:
+
+  backend:
+    build: ./backend
+    volumes:
+      - ./backend:/backend
+    expose:
+      - 8000
+
+  frontend:
+    stdin_open: true
+    build: ./frontend
+    volumes:
+      - ./frontend:/usr/src/app
+      - /usr/src/app/node_modules
+    expose:
+      - 3000
+    environment:
+      - NODE_ENV=development
```

Comment:
Do we want to hardcode the env to be development here? Feels unexpected if you are trying to run this project in production?